build.cmd

```diff
@@ -415,7 +425,7 @@ if %__BuildNative% EQU 1 (
     set __MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!
     set __MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!
 
-    @call %__ProjectDir%\run.cmd build -Project=%__IntermediatesDir%\install.vcxproj -MsBuildLog=!__MsbuildLog! -MsBuildWrn=!__MsbuildWrn! -MsBuildErr=!__MsbuildErr! -configuration=%__BuildType% %nativePlatfromArgs% %__RunArgs% %__UnprocessedBuildArgs%
+    @call %__ProjectDir%\run.cmd build -Project=%__IntermediatesDir%\install.vcxproj -MsBuildLog=!__MsbuildLog! -MsBuildWrn=!__MsbuildWrn! -MsBuildErr=!__MsbuildErr! -configuration=%__BuildType% %nativePlatfromArgs% %__RunArgs% -ExtraParameters="/p:ForceImportBeforeCppTargets=%__ProjectDir%/clr.nativebuild.props /m:2" %__UnprocessedBuildArgs%
```

Comment:
Any reason you are passing this as a parameter instead of just including clr.nativebuild.props via dir.props? Actually thinking about that I'm not entirely sure dir.props gets imported, but perhaps we can tell cmake to import it in one of the cmake files instead of having to remember to pass this extra property everywhere we do a root native build.

Reply:
Actually it is a solution which did work. I admit that I have not tried all possible inclusion points due to time burden of working in 15 min code - test cycle. If there is a simpler solution to get this properties set for all native projects why not use it. Unfortunately CMake is not an area where I feel very comfortable.